tests/conftest.py

```diff
@@ -325,6 +325,13 @@ def cluster_teardown_finalizer():
     # Verify health of ceph cluster
     # TODO: move destroy cluster logic to new CLI usage pattern?
     log.info("Done creating rook resources, waiting for HEALTH_OK")
+    # patch gp2 (EBS) storage class as 'non-default'
+    patch = " '{\"metadata\": {\"annotations\":{\"storageclass.kubernetes.io/is-default-class\":\"false\"}}}' "
+    run_cmd(
+        f"oc patch storageclass gp2 "
+        f"-p {patch} "
+        f"--timeout=120s"
```

Comment:
looks like --timeout option is not available with oc patch. instead we can use --request-timeout

```
(venv1) [vavuthu@localhost ocs-ci]$ oc patch storageclass gp2 -p  '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}' --timeout=120s
Error: unknown flag: --timeout

venv1) [vavuthu@localhost ocs-ci]$ oc patch storageclass gp2 -p  '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}' --request-timeout="120s"
storageclass.storage.k8s.io/gp2 patched
(venv1) [vavuthu@localhost ocs-ci]$ 

```

Reply:
Thanks fixing it!